app/src/main/java/com/github/mateo762/myapplication/ui/home/SummaryScreen.kt

```diff
@@ -30,7 +31,7 @@ fun HabitListScreen(habits: List<Habit>) {
                     context.startActivity(intent)
                 },
                 backgroundColor = Color.White,
-                contentColor = Color.Black
+                contentColor = colorResource(R.color.today_background)
```

Comment:
This change looks a bit weird, the background is usually bright. If it is intentional, and you checked, then it is fine

Reply:
It is a slightly dark color for the + sign in the FAB